src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -303,9 +300,17 @@ public IColony getColonyByPosFromDim(final RegistryKey<World> registryKey, @NotN
     @Override
     public boolean isTooCloseToColony(@NotNull final World w, @NotNull final BlockPos pos)
```

Comment:
this method name is confusing, we should maybe make this return true, remove the ! in the lower one, and then adjust the handling ones too, while on it xD